packages/@lwc/engine-core/src/framework/def.ts

```diff
@@ -254,7 +254,7 @@ export function getComponentInternalDef(Ctor: unknown, name?: string): Component
 
 // Only set prototype for public methods and properties
 // No DOM Patching occurs here
-export function setElementProto(elm: Element, def: ComponentDef) {
```

Comment:
Any reason to use a generic here? It is not being referenced anywhere.

Reply:
The `HostElement` generic is needed to remove the `Element` type of the `elm` argument.

Reply:
That's a good point @ravijayaramappa. I don't think we need the generic here since it is not used in the function or in the retuned type.

Reply:
Ah, I see what you meant!